<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleFiles</class>
 <widget class="QWidget" name="MultipleFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert Multiple Files</string>
  </property>
  <widget class="QComboBox" name="fileT">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>.g3db</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.g3dj</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.fbx</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="pack">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pack Colors</string>
   </property>
  </widget>
  <widget class="QPushButton" name="convert">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_folder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxV">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="value">
    <number>32000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Folder:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeWindow">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>Single File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Folder:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxW">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="log">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Log</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Bones:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_folder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Vertices:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>File Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Weights:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="flip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip V</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
  <widget class="QPushButton" name="browse0">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
